cmd/md2html/main.go

```diff
@@ -296,7 +296,7 @@ func writeCode(w io.Writer, path, snippet string) {
 func readSnippet(path, snippet string) (string, error) {
 	b, err := ioutil.ReadFile(path)
 	if err != nil {
-		return "", fmt.Errorf("Unable to read source file: %s\n", path)
+		return "", fmt.Errorf("Unable to read source file: %s", path)
```

Comment:
While you're at it, how about s/Unable/unable/?

https://github.com/golang/go/wiki/CodeReviewComments#error-strings